app/posts/manage-breast-screening/2025/06/2025-06-09-recording-previous-mammograms.md

```diff
@@ -0,0 +1,142 @@
+---
+  title: Recording previous mammograms
+  description: How we're helping radiographers and breast clinic admin teams to log mammograms that aren’t on their system
+  date: 2025-06-09
+  tags:
+    - alpha
+    - prototype
+    - breast screening
+---
+
+We’re helping radiographers and breast clinic admin teams to log mammograms that aren’t on their system. This will help radiologists when reviewing breast images, and protect participants from unnecessary radiation exposure by having X-rays too soon. 
+
+## Getting a full picture 
+
+In routine breast cancer screening, women are invited to have a mammogram every three years. They have their breasts X-rayed, and the images taken are assessed by a specialist.  
+
+The X-rays are stored for the future so radiologists looking for signs of cancer can compare the latest images against previous scans. They refer to them to see if something that causes them concern has appeared previously, or has already been investigated. This will influence their decision on whether to recall a participant for further assessment. 
```

Comment:
```suggestion
The X-rays are stored for the future so radiologists looking for signs of cancer can compare the latest images against previous scans. They refer to them to see if something that causes them concern has appeared previously, or has already been investigated. This will influence their decision on whether to recall a participant for further assessment. 
```
```suggestion
The X-rays are stored for the future so radiologists looking for signs of cancer can compare the latest images against previous scans. They refer to them to see if something that causes them concern has appeared previously, or has already been investigated. This can influence their decision on whether to recall a participant for further assessment. 
```